### ISSUE TITLE
New autofail attribute: Raw Goods

### ISSUE THREAD

**Issue by TCHalogen** [flashflashrevolution/rCubed#23]
For players who are trying to actually improve their level ranks, the in-game autofail doesn't serve very much use since the values operate in parallel. For example, if a player is trying to improve from a 4 good score, they can set their autofail's Good count to 4 goods, but nothing stops them from getting 3 averages, or 2 misses (or both).

Adding a raw goods attribute will allow for all combinations of judgments used to produce a worse score to kill off a run.

Optional implementation: For more ease of use, either a single parsed text box with hyphen-delimited values (e.g. 4-2-0-0) can calculate the raw good counts in the event that the user knows their PA but not the raw goods off the top of their head, or a row of four smaller boxes with inputs for goods/averages/boos/misses can be used to aggregate the totals.

**Comment by mat100payette** [comment]
Just as a bonus (that we've discussed briefly), it would be really nice to have some sort of checkbox for autofail that ignores all other autofail inputs and takes your PB (Personal Best) in raw goods as the autofail value. That way, a player wouldn't have to manually input their PB for every song when going for new best scores.

**Comment by TCHalogen** [comment]
Can have that added as a separate enhancement; I like the idea.

**Comment by G-flat** [comment]
@all-contributors please add @TCHalogen for idea

**Comment by G-flat** [comment]
@all-contributors please add @TCHalogen for ideas

**Comment by G-flat** [comment]
@all-contributors please add @TCHalogen for ideas and code
